.env

```diff
@@ -0,0 +1 @@
+BONSAI_URL=https://yb89f43s:ounkgiruq9y7rfei@holly-4384942.us-east-1.bonsaisearch.net
```

Comment:
we should remove `.env` and add it to `.gitignore` -- because we don't want people to have access to our DB.